testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusSiteTest.java

```diff
@@ -106,31 +111,22 @@ public void validateQuarkusVersionMatch(TestInfo testInfo) throws Exception{
             throw new UncheckedIOException(e);
         }
 
-        final HtmlPage page = loadPage(webPageUrl, 60);
+        Page page = loadPage(webPageUrl, 60);
         LOGGER.info("Trying to find element: " + elementQuarkusPlatformVersionByXpath);
-        String quarkusPlatformVersionFromWeb = page.getFirstByXPath(elementQuarkusPlatformVersionByXpath);
+        String quarkusPlatformVersionFromWeb = page.locator(elementQuarkusPlatformVersionByXpath).elementHandle().getAttribute("title");
 
         assertNotNull(quarkusPlatformVersionFromWeb, "Element: " + elementQuarkusPlatformVersionByXpath + " is missing!");
         assertTrue(quarkusPlatformVersionFromWeb.contains(quarkusPlatformVersion),
                 "Quarkus versions doesn't match. Found on the web: " + quarkusPlatformVersionFromWeb + ". Expected: " + quarkusPlatformVersion);
     }
 
 
-    public HtmlPage loadPage(String url, int timeoutSeconds) throws InterruptedException {
-        HtmlPage page = null;
-        long startTime = System.currentTimeMillis();
-        long endTime = startTime + timeoutSeconds * 1000;
+    public Page loadPage(String url, int timeoutSeconds) {
         LOGGER.info("Loading web page " + url);
-        try {
-            page = webClient.getPage(url);
-        } catch (Exception e) {
-            throw new RuntimeException("Cannot load the page: " + webPageUrl);
-        }
-        JavaScriptJobManager manager = page.getEnclosingWindow().getJobManager();
-        while (manager.getJobCount() > 0 && System.currentTimeMillis() < endTime) {
-            Thread.sleep(1000);
-        }
-
+        Page page = browserContext.newPage(); // this will create new tab inside browser
+        page.setDefaultTimeout(10 * 1000);
```

Comment:
```suggestion
        page.setDefaultTimeout(timeoutSeconds * 1000);
```
The original timeout was `timeoutSeconds * 1000`, with 1000ms sleeps.

Reply:
good one